src/quo/components/wallet/transaction_progress/view.cljs

```diff
@@ -0,0 +1,230 @@
+(ns quo.components.wallet.transaction-progress.view
+  (:require [quo.components.buttons.button.view :as button]
+            [quo.components.icon :as icons]
+            [quo.components.markdown.text :as text]
+            [quo.components.tags.context-tag.view :as context-tag]
+            [quo.components.wallet.progress-bar.view :as progress-box]
+            [quo.components.wallet.transaction-progress.style :as style]
+            [quo.foundations.colors :as colors]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            [utils.i18n :as i18n]))
+
+(def arbitrum-finalising-progress-percentage 26)
+(def optimism-finalising-progress-percentage 16)
```

Comment:
This can be a configurable prop in the component, IMO the component should abstract from the app domain variables

Reply:
Done